surveyor/collector_statz.go

```diff
@@ -599,14 +602,10 @@ func NewStatzCollector(nc *nats.Conn, logger *logrus.Logger, numServers int, ser
 		jsServerLabels:     []string{"server_id", "server_name", "cluster_name"},
 		jsServerInfoLabels: []string{"server_name", "server_host", "server_id", "server_cluster", "server_domain", "server_version", "server_jetstream"},
 		constLabels:        constLabels,
-		jszFilterList:      make(map[string]struct{}),
+		jszFilterSet:       make(map[JszFilter]bool),
```

Comment:
empty struct{} more idiomatic here IMO

Reply:
So, empty struct takes less memory(irrelevant here), but having bool makes the code that checks for value nicer/shorter